### PR TITLE
Axes, Growing and shrinking, and Introduction to Flexbox : Move scrimbs.

### PR DESCRIPTION
**Pull request by SohamKukreti**
## Because

The purpose of this PR is to shift the existing Scrimba embeds in lessons, from their current position to Additional Resources as link text instead of embedded to fix [this issue](https://github.com/TheOdinProject/curriculum/issues/26057).

## This PR

* Moves existing Scrimba embed in [Introduction to Flexbox](https://www.theodinproject.com/lessons/foundations-introduction-to-flexbox) from its original position to Additional Resources.
* Moves two existing Scrimba embeds in [Growing and Shrinking](https://www.theodinproject.com/lessons/foundations-growing-and-shrinking) from their original position to Additional Resources.
*  Moves existing Scrimba embed in [Axes](https://www.theodinproject.com/lessons/foundations-axes) from its original position to Additional Resources.

## Issue

Related to #26057 

## Pull Request Requirements

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`
-   [x] The `Because` section summarizes the reason for this PR
-   [x] The `This PR` section has a bullet point list describing the changes in this PR
-   [x] If this PR addresses an open issue, it is linked in the `Issue` section
-   [x] If any lesson files are included in this PR, they have been previewed with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure it is formatted correctly
-   [x] If any lesson files are included in this PR, they follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

### CODE REVIEW

File: foundations/html_css/flexbox/flexbox-axes.md [hunk 1]
```diff
@@ -62,3 +59,5 @@ This section contains questions for you to check your understanding of this less
 This section contains helpful links to related content. It isn’t required, so consider it supplemental.
 
 *   [This flexbox visual cheatsheet](https://flexbox.malven.co/) has some useful references to flex and its properties.
+
+* For an interactive demo of how axes work with flexbox, check out [this Scrim](https://scrimba.com/learn/flexbox/main-axis-and-cross-axis-flexbox-tutorial-cz94MT8).
```
Comment (thatblindgeye) [inline foundations/html_css/flexbox/flexbox-axes.md:63]:
```suggestion
* For an interactive demo, check out this [Scrim on Flexbox axes](https://scrimba.com/learn/flexbox/main-axis-and-cross-axis-flexbox-tutorial-cz94MT8). Note that this Scrim requires logging into Scrimba in order to view.
```

I'm noticing the needing to login issue. Can you confirm whether you get that too, @SohamKukreti ?

File: foundations/html_css/flexbox/flexbox-growing-and-shrinking.md [hunk 2]
```diff
@@ -103,3 +94,7 @@ This section contains questions for you to check your understanding of this less
 This section contains helpful links to related content. It isn’t required, so consider it supplemental.
 
 *  [Check out this video](https://www.youtube.com/watch?v=u044iM9xsWU&t=1s&pp=ugMICgJhchABGAE%3D) explaining how flexbox works and why.
+
+* For an interactive explanation and demo of the flex shorthand, check out [this Scrim](https://scrimba.com/learn/flexbox/the-flex-property-flexbox-tutorial-cGNKJTv).
+
+* Watch [this interactive Scrim](https://scrimba.com/learn/flexbox/flex-grow-shrink-basis-flexbox-tutorial-ck6L7fv) for an alternative explanation and demo of using flex-grow, flex-shrink and flex-basis in a real world scenario.
```
Comment (thatblindgeye) [inline foundations/html_css/flexbox/flexbox-growing-and-shrinking.md:100]:
```suggestion
* For an interactive explanation and demo, check out the [Scrim on the flex shorthand](https://scrimba.com/learn/flexbox/the-flex-property-flexbox-tutorial-cGNKJTv). For an alternative explanation you can view the [Scrim on using flex-grow, flex-shrink, and flex-basis](https://scrimba.com/learn/flexbox/flex-grow-shrink-basis-flexbox-tutorial-ck6L7fv). Note that these Scrims require logging into Scrimba in order to view.
```

File: foundations/html_css/flexbox/flexbox-intro.md [hunk 3]
```diff
@@ -88,3 +86,5 @@ This section contains questions for you to check your understanding of this less
 This section contains helpful links to related content. It isn’t required, so consider it supplemental.
 
 * [This Flexbox tutorial](https://internetingishard.netlify.app/html-and-css/flexbox/index.html) is a friendly tutorial for modern CSS layouts by Interneting Is Hard.
+
+* For a more interactive explanation and example on Flexbox, try [this Scrim](https://scrimba.com/learn/flexbox/your-first-flexbox-layout-flexbox-tutorial-canLGCw).
```
Comment (thatblindgeye) [inline foundations/html_css/flexbox/flexbox-intro.md:90]:
```suggestion
* For a more interactive explanation and example, try this [Scrim on Flexbox](https://scrimba.com/learn/flexbox/your-first-flexbox-layout-flexbox-tutorial-canLGCw). Note that this Scrim requires logging into Scrimba in order to view.
```

**Review by thatblindgeye — CHANGES_REQUESTED** [review]
Thanks for the updates! Had some changes below. When adding link text, we need to avoid being too vague. "This article", "click here", etc are very difficult for users navigating with assistive tech (like screen readers) to understand, especially if there are multiple links with the same link text.

File: foundations/html_css/flexbox/flexbox-axes.md (hunk 1 above)
Comment (SohamKukreti) [reply foundations/html_css/flexbox/flexbox-axes.md:63]: Yeah I get the login prompt as well. I suppose scrimba has change some of their policies recently

**Review by thatblindgeye — APPROVED** [review]

**Pull request merged**